manifests/g/GitHub/GitHubDesktop/Beta/3.4.1-beta1/GitHub.GitHubDesktop.Beta.locale.en-US.yaml

```diff
@@ -0,0 +1,28 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: GitHub.GitHubDesktop.Beta
+PackageVersion: 3.3.19-beta2

```

Comment:
```suggestion
PackageVersion: 3.3.19-beta1
```

Reply:
Fixed. Copy paste error of myself as there's currently a bug in the automatic pipeline